### DISCUSSION TITLE
Project status

### DISCUSSION THREAD

**Discussion by epilys** [epilys/gerb#44] [Announcements]
I have stopped daily development for some time now, due to receiving many discouraging comments about the project. I don't know if I want to continue if the only outcome is small doses of negativity. I'm not shutting it down though, so this is not necessarily a end.

**Comment by moan0s** [comment]
Mhh sad to hear that. Thanks for all the work!

**Comment by alerque** [comment]
I've been a bit out of the font development loop for a while with my energy being sucked up elsewhere, but I've been following along on progress with interest.

Of course your time choices are your own, but don't give up *just* because of negativity. FOSS dev work is a kind of thankless investment of time—with some bright exceptions now and then usually negativity means people are interested in what you are doing (enough to keep trying it). Most folks are either missing the skill to contribute or the sense to be positive about the dev process of stuff they are using for free (or both). That in itself isn't a measure of value of the project.

There may or may not be other reasons to pursue it or not, but don't let *just* that issue get you down.

**Comment by johnblommers** [comment]
Do not let negative commenters get you down. 

I think many people are likely to write "Your product A sucks because it can't do X," instead of writing "I liked A a lot. Might it also do X at some future time. Here's why I think it's a good idea..."

Even the good Linus Torvalds got out of hand with his powerful criticism and name-calling over technical issues he objected to.

Remember why you started this project and focus on that for your inspiration and perspiration.

Best wishes to you!
